content/sigs/docs/gsod/2021/organization-proposal.adoc

```diff
@@ -36,15 +34,63 @@ The improved documentation will also be referenced in a Kubernetes Solutions Pag
 
 === Your project's problem
 
-More details to be inserted here...
+Jenkins users have significant challenges deploying on Kubernetes.
+A successful Jenkins deployment on Kubernetes combines many different Jenkins components.
+Mistakes in using those components create a less reliable and less usable Jenkins installation.
+
+There are many levels to be addressed and many places where incomplete information and mistakes lead to installation and upgrade problems.
+The Jenkins documentation created by this project will describe the compromises and the alternatives for Jenkins on Kubernetes.
+Documentation users will be able to confidently deploy Jenkins on Kubernetes with an understanding of the strengths and weaknesses of their solution.
 
 === Your project's scope
 
-More details to be inserted here...
+Project scope will be adapted based on available time from the writer.
+
+==== Preparation Phase
+
+* Review link:https://leanpub.com/the-devops-2-3-toolkit[DevOps 2.3 Toolkit] by Viktor Farcic
+* Review and explore link:https://leanpub.com/the-devops-2-4-toolkit[DevOps 2.4 Toolkit] by Viktor Farcic (open source, content can be reused with attribution)
+* Review and explore link:/doc/book/installing/kubernetes/[Installing Jenkins - Kubernetes], identify and fix issues detected
+
+==== Writing Phase 1
+
+* Document creating a Docker image that contains plugins and use it in Kubernetes
+* Document maintaining plugin versions with plugin installation manager tool
+
+==== Writing Phase 2
+
+* Document maintaining system configuration with configuration as code
+
+==== Writing Phase 3
+
+* Document ephemeral agents in a Kubernetes cluster
+** Using the kubernetes plugin for ephemeral agents
+** Using static agents
+
+==== Writing Phase 5
```

Comment:
Phase 4 is missing. Too many phases also imho

Reply:
Thanks.  Fixed in 354396873faf3faf7c80ff0ee6e89d0af257ef6a